src/components/Home/HomeHeader.js

```diff
@@ -0,0 +1,133 @@
+import { useState } from 'react'
+import { Link } from "react-router-dom"
+import { Header, Icon, Text, Button, NavBar } from 'components/Reusable'
+import { useAuth, useTheme } from "contexts"
+import { getTextColor, getIconColor, getBgColor } from 'utils'
+
+const HomeHeader = () => {
+    const [isSmallNavVisible, setIsSmallNavVisible] = useState(false)
+    const { theme, toggleTheme } = useTheme()
+    const { isUserLoggedIn, logoutUser } = useAuth()
+
+    function toggleNavVisibility() {
+        setIsSmallNavVisible(!isSmallNavVisible)
+    }
+
+    return (
+        <Header>
+
+            <Link to='/'>
+                <Text classes='txt-lg txt-ucase'>
+                    sneakerviews
+                </Text>
+            </Link>
+
+            <NavBar id='nav-big' classes='flx-min-center'>
+
+                <Link to='/videos'>
+                    <Button classes={`btn-txt ${getTextColor(theme)} txt-md txt-lcase mg-right-lg`}>
+                        browse
+                    </Button>
+                </Link>
+
+                <Link to='/playlists'>
+                    <Icon classes={`${getIconColor(theme)} mg-right-lg`}>
+                        playlist_play
+                    </Icon>
+                </Link>
+
+
+                <Link to='/history'>
+                    <Icon classes={`${getIconColor(theme)} mg-right-lg`}>
+                        history
+                    </Icon>
+                </Link>
+
+                {
+                    isUserLoggedIn
+                        ? <Button onClick={logoutUser} classes={`btn-txt txt-lcase ${getTextColor(theme)} pd-xs txt-md`}>
+                            logout
+                        </Button>
+                        : <Link to='/login'>
+                            <Button classes={`btn-txt txt-lcase ${getTextColor(theme)} pd-xs txt-md`}>
+                                login
+                            </Button>
+                        </Link>
+                }
+
+            </NavBar>
+
+            <div className='pos-relative'>
+
+                <Button id='btn-ham' onClick={toggleNavVisibility} classes='btn-txt mg-top-xs'>
+
+                    <Icon classes={getIconColor(theme)}>
+                        menu
+                    </Icon>
+
+                </Button>
+
+                {
+                    isSmallNavVisible &&
+                    <NavBar id='nav-small' classes={`flx-column pd-md ${getBgColor(theme)} pos-absolute z-index tl-0 b-solid`}>
+
+                        <Button onClick={toggleNavVisibility} classes='btn-txt mg-btm-xs'>
```

Comment:
Instead of classes , we should be using className

Reply:
I have built these reusable components that accept `classes` argument for `className`.